Pod/Classes/URBNAlertView.m

```diff
@@ -111,12 +119,14 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         NSDictionary *metrics = @{@"sectionMargin" : self.alertStyler.sectionVerticalMargin,
                                   @"btnH" : self.alertStyler.buttonHeight,
                                   @"lblHMargin" : self.alertStyler.labelHorizontalMargin,
-                                  @"titleVMargin" : titleMargin,
+                                  @"topVMargin" : @([titleMargin doubleValue] + [self.alertStyler.topMargin doubleValue]),
```

Comment:
Native `UIAlertController` view has larger top margins than margins between sections. So. it's impossible to layout that with a single parameter (`sectionVerticalMargin`). Maybe it would be reasonable to use `titleMargin` for that, but for backward compatibility I've added a new parameter. Also as for me it makes solid picture - every element has its own margin and also there is a general margin for all content in the view.